tests/unit/test_extract_email.py

```diff
@@ -35,6 +35,9 @@ def test_extract_email_disabled(urlextract, text, expected):
     ("<email@address.net>",
      ['email@address.net']),
 
+    ("email with comma ema,il@address.net",
+     []),
```

Comment:
I am not sure if it should return empty list. 
I think about returning something like: `[il@address.net]` 

What is your opinion?